logicmonitor_data_sdk/api/logs.py

```diff
@@ -165,9 +167,18 @@ def _do_request(self):
       self.UnLock()
 
   def _merge_request(self, single_request):
+    # size limiting
+    payload_cache = json.dumps(self._payload_cache)
+    compressed_payload_cache = gzip.compress(payload_cache.encode("utf-8"))
+    serialized_single_request = self.api_client.sanitize_for_serialization(single_request)
+    single_request_json = json.dumps(serialized_single_request)
+    compressed_single_request = gzip.compress(single_request_json.encode("utf-8"))
+    if (compressed_payload_cache.__sizeof__() + compressed_single_request.__sizeof__() > 104858) or \
```

Comment:
It would be better to put 104858 , and 1048576 in a constant. It would be easier to maintain later if compression size limit changes.

Reply:
Introduced constants for size limiting.